jdtls.ext/com.microsoft.java.maven.plugin/src/main/java/com/microsoft/java/maven/ArtifactSearcher.java

```diff
@@ -92,13 +86,36 @@ public static void initialize(String path){
         }
     }
 
+    public static List<ArtifactResult> searchByGA(String groupId, String artifactId, IProgressMonitor monitor) {
```

Comment:
`GA` is not human-readable. consider `Identifier`

Reply:
ok.